### PR TITLE
Remove cosmetic objects from Torvus Temple

### PR DESCRIPTION
**Pull request by henriquegemignani**

### CODE REVIEW

File: open_prime_rando/echoes/specific_area_patches.py [hunk 1]
```diff
@@ -67,3 +70,27 @@ def torvus_generator(editor: PatcherEditor):
...
+        first_pass.remove_instance(obj)
+
+    for obj in ["GENERATE GIBS"]:
+        second_pass.remove_instance(obj)
+
+    # These are generated objects
+    # debris = ["SwampCrateDebris"] * 7
+    # for obj in ["GibFlash", "Sound - Swamp Crate Gib"] + debris:
+    #     default.remove_instance(obj)
+
```
Comment (duncathan) [inline open_prime_rando/echoes/specific_area_patches.py:96]: ?

Comment (henriquegemignani) [reply open_prime_rando/echoes/specific_area_patches.py:96]: These are things I wanted to remove as well, but they're generated objects.

Comment (duncathan) [reply open_prime_rando/echoes/specific_area_patches.py:96]: gotcha. I assume to safely remove them we'd need to do something more involved?

Comment (henriquegemignani) [reply open_prime_rando/echoes/specific_area_patches.py:97]: Updated the comment. Just add some nice wrapper for the generated objects section/layer/whatever to RDS.

**Pull request merged**